ros_gz_bridge/src/convert/sensor_msgs.cpp

```diff
@@ -288,6 +298,30 @@ convert_gz_to_ros(
   convert_gz_to_ros(gz_msg.linear_acceleration(), ros_msg.linear_acceleration);
 
   // Covariances not supported in gz::msgs::IMU
+  int data_size = gz_msg.linear_acceleration_covariance().data_size();
+  if (data_size == 9)
```

Comment:
can you define 9 as a const?

Reply:
again why a const instead of a simple literal? IMO, introducing a variable is not worth it in this context.
What do you think ? @ahcorde